### ISSUE TITLE
Run LFQanalyst on GA with demo test data

### ISSUE THREAD

**Issue by supernord** [usegalaxy-au/tools-au#100]
LFQanalyst should be able to run with its included demo data - is there any way to accommodate this?
Users who expect this functionality may think the tool is broken on GA when they try to run it.

**Comment by neoformit** [comment]
Just to clarify are you saying there is an error when you run it with its test data?

**Comment by supernord** [comment]
It seems that an interactive tool will only start with user provided data.
Maybe there is no way around this?

**Comment by neoformit** [comment]
Yep, I guess "launch with test data" could be an option, if you really think that would be valuable. @TomHarrop ?

**Comment by supernord** [comment]
I think it would be valuable as this is how LFQanalyst is expected to run. When you first use the tool, there is a demo with test data so that you can see how the tool works before uploading your own data.

**Comment by TomHarrop** [comment]
No worries, it bundles the test data. I will do this when I get back from leave.
